.gitignore

```diff
@@ -1 +1,2 @@
 /target
+.vscode
```

Comment:
wait, just saw this now. Please split in a separate commit (not PR, just commit) as it's completely unrelated to this commit.

Reply:
```sh
git reset HEAD^
git add -p # Only choose the .gitignore change
git commit 
git commit -a -C 05fcddb8e3b05140bec84c8f435c1e6fa53e5251
```

Reply:
Done

Reply:
Ah ok, coming...